netlify.toml

```diff
@@ -1,8 +1,3 @@
-
-[build.environment]
-  NPM_VERSION = "7.9.0"
-  NODE_VERSION = "15.0.1"
-
```

Comment:
This section explains the variables
https://docs.netlify.com/configure-builds/environment-variables/#netlify-configuration-variables

These docs explain the focal default. Which mention default is Node 16.
https://github.com/netlify/build-image/blob/focal/included_software.md

Reply:
I triggered another deploy preview by clearing cache and using the last commit.
The log error was here:
'
Interesting that it still mentioned Node 12.

```
2:44:20 PM: build-image version: b3298a4e6935499b0fd0c6cd06bb13d1bb899d1a (focal)
2:44:20 PM: buildbot version: b3298a4e6935499b0fd0c6cd06bb13d1bb899d1a
2:44:20 PM: Building without cache
2:44:20 PM: Starting to prepare the repo for build
2:44:20 PM: No cached dependencies found. Cloning fresh repo
2:44:20 PM: git clone --filter=blob:none https://github.com/rackerlabs/docs-cloud-load-balancers
2:44:21 PM: Preparing Git Reference pull/312/head
2:44:21 PM: Parsing package.json dependencies
2:44:22 PM: Starting build script
2:44:22 PM: Installing dependencies
2:44:22 PM: Python version set to 3.8
2:44:23 PM: Downloading and installing node v12.18.0...
2:44:23 PM: Downloading https://nodejs.org/dist/v12.18.0/node-v12.18.0-linux-x64.tar.xz...
2:44:23 PM: Computing checksum with sha256sum
2:44:23 PM: Checksums matched!
2:44:24 PM: Now using node v12.18.0 (npm v6.14.4)
2:44:24 PM: Started restoring cached build plugins
2:44:24 PM: Finished restoring cached build plugins
2:44:25 PM: Attempting Ruby version 2.7.1, read from environment
2:44:25 PM: Required ruby-2.7.1 is not installed - installing.
2:44:25 PM: Searching for binary rubies, this might take some time.
```

Reply:
netlify seems to want node 16, so I went ahead and put variables back in place and it actually updated the node installation in the logs. So probably best to keep them in.
```




3:15:32 PM: build-image version: b3298a4e6935499b0fd0c6cd06bb13d1bb899d1a (focal)
3:15:32 PM: buildbot version: b3298a4e6935499b0fd0c6cd06bb13d1bb899d1a
3:15:32 PM: Fetching cached dependencies
3:15:32 PM: Failed to fetch cache, continuing with build
3:15:32 PM: Starting to prepare the repo for build
3:15:33 PM: No cached dependencies found. Cloning fresh repo
3:15:33 PM: git clone --filter=blob:none https://github.com/rackerlabs/docs-cloud-load-balancers
3:15:33 PM: Preparing Git Reference pull/312/head
3:15:33 PM: Parsing package.json dependencies
3:15:34 PM: Starting build script
3:15:34 PM: Installing dependencies
3:15:34 PM: Python version set to 3.8
3:15:35 PM: v16.19.0 is already installed.
3:15:35 PM: Now using node v16.19.0 (npm v8.19.3)
3:15:35 PM: Enabling Node.js Corepack
3:15:35 PM: Started restoring cached build plugins
3:15:35 PM: Finished restoring cached build plugins
3:15:35 PM: Attempting Ruby version 2.7.1, read from environment
3:15:36 PM: Required ruby-2.7.1 is not installed - installing.
3:15:36 PM: Searching for binary rubies, this might take some time.
3:15:36 PM: Found remote file https://rvm_io.global.ssl.fastly.net/binaries/ubuntu/20.04/x86_64/ruby-2.7.1.tar.bz2
3:15:36 PM: Checking requirements for ubuntu.
3:15:36 PM: Requirements installation successful.
```